README.md

```diff
@@ -115,6 +115,10 @@ The complete cuSignal API documentation including a complete list of functionali
 [cuSignal 0.15 API](https://docs.rapids.ai/api/cusignal/stable/) | [cuSignal 0.16 Nightly](https://docs.rapids.ai/api/cusignal/nightly/)
 
 ## Installation
+### Note: cuSignal installs through Conda will run natively on **ALL** supported GPU architectures.
+
+1. CUDA 10.X -- [50, 52, 53] (Maxwell), [60, 61, 62] (Pascal), [70, 72] (Volta), [75] (Turing)
+2. CUDA 11.X -- [50, 52, 53] (Maxwell), [60, 61, 62] (Pascal), [70, 72] (Volta), [75] (Turing), [80] (Ampere)
```

Comment:
Let's remove this section. I know what you're trying to say here, but I think the message gets muddled a bit. The big goal, for me, is to:

- Say that Conda is the preferred installation mechanism, and if you use Conda, don't worry about your CC unless you're before Maxwell. Additionally, if you're on Ampere, you'll have to use CUDA 11

Let's address these fine points in a README overhaul. It's probably time for me to do an update anyway :).

Reply:
Remove with no replacement?

Reply:
Back to original